.env

```diff
@@ -1,4 +0,0 @@
-MSSQL_CLI_SERVER=
```

Comment:
We still want this file to be checked in so that devs know what variables to set.